client/app/templates/map.hbs

```diff
@@ -12,6 +12,7 @@
     {{#unless showingAllStudents}}
       <div class='msg msg--map'>
         {{fa-icon 'warning'}}
+        <i class='fa fa-close close' onClick={{action 'closeAlert'}}></i>
```

Comment:
I believe `<i class='fa fa-close close' {{action 'closeAlert'}}></i>` would be sufficient here, since a click is the default event that triggers an action.